### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by ShiroYaksha90**
## Hi 👋, in this pull request I:

- Implemented the design from the sneak peek wireframes.
- Used methods that I have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,11 @@
-<h1>Hello, the posts will be listed here ...</h1>
\ No newline at end of file
+<%= render 'shared/header'%>
+<%= render 'shared/user-card', user: @user %>
+<div class = 'posts-container'><%@user.recent_three_posts.each do |post|%>
```
Comment (VICTOR-CHUKWU) [inline app/views/posts/index.html.erb:3]: - [ ] from the design and project requirement, posts should display the 5 most recent comment, currently your app is displaying all comments here, kindly fix this
design :point_down: 
Screenshot from 2023-04-06 17-35-52
your design :point_down: 
Screenshot from 2023-07-22 02-08-21

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @ShiroYaksha90  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShiroYaksha90** [comment]
@VICTOR-CHUKWU Thank you for the review, I'll implement the required changes

**Review by Munsa1 — APPROVED** [review]
Hi @ShiroYaksha90, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made Requested Change By Previous Reviewer :heavy_check_mark: 
 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShiroYaksha90** [comment]
@Munsa1 Thank you very much

**Pull request merged**